db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
+      t.string :host_name
+      t.string :status
+      t.string :scheduling_status
+      t.string :zone_name
+      t.string :disabled_reason
```

Comment:
disabled_reason is connected to scheduling_status. Scheduling status can be either 'enabled' or 'disabled' - maybe I should make it boolean now and rename disabled_reason to schedulind_disabled_reason.

I named 'scheduling_status' in this way even Fog uses 'status' for it b/c Fog also has 'state' attribute which contains information whether Service is running or not - the same information as we store in SystemService as 'status' so I wanted to have coherent naming among miq models.